pom.xml

```diff
@@ -77,8 +77,8 @@
 				<artifactId>maven-compiler-plugin</artifactId>
 				<version>3.3</version>
 				<configuration>
-					<source>1.7</source>
-					<target>1.7</target>
+					<source>1.8</source>
```

Comment:
Could use https://github.com/jenkinsci/pom and get rid of the custom stuff. Not important for now